main.go

```diff
@@ -58,6 +59,11 @@ func main() {
 			EnvVar:      "CHANNEL_SERVER_VERSION",
 			Destination: &ChannelServerVersion,
 		},
+		cli.StringFlag{
+			Name:        "path-prefix",
+			EnvVar:      "PATH_PREFIX",
+			Destination: &PathPrefix,
+		},
```

Comment:
Provide a default path prefix equivalent to the old hard-coded value.
```suggestion
		cli.StringFlag{
			Name:        "path-prefix",
			EnvVar:      "PATH_PREFIX",
			Value:       "v1-release",
			Destination: &PathPrefix,
		},
```